winpr/libwinpr/utils/wlog/wlog.c

```diff
@@ -346,7 +346,8 @@ static BOOL WLog_WritePacket(wLog* log, wLogMessage* message)
 }
 
 BOOL WLog_PrintMessageVA(wLog* log, DWORD type, DWORD level, DWORD line,
-                         const char* file, const char* function, va_list args)
+                         const char* file, const char* function,
```

Comment:
Why this change?

Reply:
To be able to call this function from the scquery.c report_level function that has a format string and a va_list arguments.
The format argument to WLog_PrintMessageVA is mandatory, while we could pass no argument for the va_list (it can be empty). Therefore it is safer to explicitely define it.